addons/spreadsheet/static/tests/utils/data.js

```diff
@@ -408,6 +408,19 @@ export function getBasicData() {
                     currency_id: 2,
                     pognon: 1000,
                 },
+                {
```

Comment:
I don't think I'd add a new record in the test data. I'd give it a 87% of chance that it'll break a test, somewhere, in some version. Maybe just add the record for your test ?

Reply:
apparently it already breaks everything, i'll do that